mod_reforged/hooks/entity/world/attached_location/fortified_outpost_location.nut

```diff
@@ -0,0 +1,38 @@
+::mods_hookExactClass("entity/world/attached_location/fortified_outpost_location", function(o) {
+	local onUpdateShopList = o.onUpdateShopList;
+	o.onUpdateShopList = function( _id, _list )
+	{
+		switch (_id)
+		{
+			case "building.marketplace":
+				if (!this.getSettlement().isSouthern())
+				{
+					_list.extend([
+						{
+							R = 50,
+							P = 1.0,
+							S = "helmets/rf_scale_helmet"
+						},
+						{
+							R = 60,
+							P = 1.0,
+							S = "helmets/rf_padded_scale_helmet"
+						},
+						{
+							R = 85,
+							P = 1.0,
+							S = "weapons/rf_battle_axe"
+						},
+						{
+							R = 85,
+							P = 1.0,
+							S = "weapons/rf_reinforced_wooden_poleflail"
```

Comment:
This addition is outside of the scope of this PR. It should be added in its own branch.